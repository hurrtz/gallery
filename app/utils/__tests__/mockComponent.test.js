import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import MockComponent from './mockComponent';

const renderer = new ShallowRenderer();

describe('<MockComponent />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<MockComponent />);

    const renderedOutput = renderer.getRenderOutput();

    expect(renderedOutput).toMatchSnapshot();
  });
});
