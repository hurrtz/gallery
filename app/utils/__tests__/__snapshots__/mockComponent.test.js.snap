// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MockComponent /> should render and match the snapshot 1`] = `
<div>
  MockComponent
</div>
`;
